### PR TITLE
Tabs template working version

### PR DESCRIPTION
**Pull request by foxleigh81**
Note: the "The content should match" test is awful but I can't work out a better way of doing this. I think I may need to sit down and pair with someone to work it out. However the test does pass so it's hopefully fit for purpose.

### CODE REVIEW

File: smoke_tests/happypath/admin_tabs-template-working-version.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Feature: Test revisions on tabs template
+
+## Done when
```
Comment (tatyree) [inline smoke_tests/happypath/admin_tabs-template-working-version.feature]: Don't leave these in the final commit, please.  The tests should be largely self explanatory.

File: smoke_tests/step_definitions/common.rb [hunk 2]
```diff
@@ -20,6 +21,11 @@
...
... could justify a change to the decision The flexible working process was not applied correctly The decision was unreasonable Many areas of MoJ have now adopted TW3 and have the option to work remotely on an occasional, voluntary basis. Guidance for remote working, either formally or as part of TW3 is given below. Specific information relating to TW3 can be found on the TW3 page. What you need to do Talk to your manager about your application Fill in the flexible working application form (FLW1a) doc Send it to your manager Try to anticipate potential issues with what you’re proposing and suggest possible solutions in your application Downloads Flexible working appeal form doc Flexible working application checklist doc Flexible working application form (FLW1a) doc Flexible working policy and guidance pdf How to request flexible working arrangements pdf Flexible working FAQs pdf How to successfully work remotely pdf How to manage remote working pdf Remote working guidance and FAQs pdf Guidance on introducing and operating a flexi-time sheme doc Job share guide Job share guide pdf View your work pattern and work-life balance pdf External sites Civil service job share network LinkedIn")
```
Comment (tatyree) [inline smoke_tests/step_definitions/common.rb]: If you want to match the entire block, which is probably *not* the case, try assigning the block to a variable using a `HERE` doc and matching against the variable.  

I would probably use a regex to match against a few keywords in roughly the correct order: 

```
expect(page).to have_text(/flexible working.+work-life.+flexible working policy/i) 
```

Comment (tatyree) [inline smoke_tests/step_definitions/common.rb]: You can pass the regex in using a variable to make the step reusable.

File: smoke_tests/happypath/admin_tabs-template-working-version.feature [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Feature: Test revisions on tabs template
+
+  Scenario: Test revisions on tabs template Acceptance Criteria
+    Given I show my environment
```
Comment (digitalronin) [inline smoke_tests/happypath/admin_tabs-template-working-version.feature]: Can this be refactored in light of #26?

Comment (tatyree) [inline smoke_tests/happypath/admin_tabs-template-working-version.feature]: Looking at #26, I agree that it probably should be.  The idea behind "I show my environment" was to have it appear once towards the start of the tests to let you know which URI the tests were running against.  It doesn't hurt anything to repeat it, but it is of little value repeating it in a full suite run.

**Comment by tatyree** [comment]
Actually, ignore that about adding the ticket.  We need to do it even if you take it out now, so I'll put the ticket in anyway.

**Comment by foxleigh81** [comment]
Created a test to check that the revisions plugin works as expected on the tabs template.

**Pull request merged**